config/src/test/java/com/redhat/lightblue/config/TestPropertyParser.java

```diff
@@ -22,13 +22,4 @@
 import com.redhat.lightblue.metadata.parser.PropertyParser;
 
 public class TestPropertyParser extends PropertyParser {
-    @Override
-    public Object parse(String name, MetadataParser p, Object node) {
-        return null;
-    }
-
-    @Override
-    public void convert(MetadataParser p, Object emptyNode, Object object) {
-
-    }
 }
```

Comment:
Extensions.getPropertyParser() returns a PropertyParser for any property
that is not recognized, and for any property for which there isn't any
registered parsers. The point of this parser is to have it in configuration
file, so we can check if the correct parser is returned for the registered
property.

On Fri, Apr 1, 2016 at 6:35 AM, Dennis Crissman <notifications@github.com>
wrote:

> In
> config/src/test/java/com/redhat/lightblue/config/TestPropertyParser.java
> <https://github.com/lightblue-platform/lightblue-core/pull/624#discussion_r58199034>
> :
>
> > @@ -22,13 +22,4 @@
> >  import com.redhat.lightblue.metadata.parser.PropertyParser;
> >
> >  public class TestPropertyParser extends PropertyParser {
> > -    @Override
> > -    public Object parse(String name, MetadataParser p, Object node) {
> > -        return null;
> > -    }
> > -
> > -    @Override
> > -    public void convert(MetadataParser p, Object emptyNode, Object object) {
> > -
> > -    }
> >  }
>
> Why have a whole class for that when you could just as easily new
> PropertyParser(){}
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/lightblue-platform/lightblue-core/pull/624/files/ea3a82c505fb57a14479315fd5b5ae19d4fdd0db#r58199034>
>